src/main/java/org/openrewrite/java/migrate/apache/commons/lang/ApacheCommonsStringUtils.java

```diff
@@ -0,0 +1,607 @@
+        }
+    }
+
+    private static class Contains {
+        @BeforeTemplate
+        boolean before(String s, String search) {
+            return StringUtils.contains(s, search);
+        }
+
+        @AfterTemplate
+        boolean after(String s, String search) {
+            return s.contains(search);
+        }
+    }
+
+    private static class CountMatches {
+        @BeforeTemplate
+        int before(String s, String pattern) {
+            return StringUtils.countMatches(s, pattern);
+        }
+
+        @AfterTemplate
+        int after(String s, String pattern) {
+            return s.length() - s.replace(pattern, "").length();
+        }
+    }
+
+    private static class DefaultString {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.defaultString(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return Objects.toString(s);
+        }
+    }
+
+    private static class DeleteWhitespace {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.deleteWhitespace(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.replaceAll("\\s+", "");
+        }
+    }
+
+    private static class EndsWithIgnoreCase {
+        @BeforeTemplate
+        boolean before(String s, String suffix) {
+            return StringUtils.endsWithIgnoreCase(s, suffix);
+        }
+
+        @AfterTemplate
+        boolean after(String s, String suffix) {
+            return s.regionMatches(true, 0, suffix, 0, suffix.length());
+        }
+    }
+
+    private static class EqualsIgnoreCase {
+        @BeforeTemplate
+        boolean before(String s, String other) {
+            return StringUtils.equalsIgnoreCase(s, other);
+        }
+
+        @AfterTemplate
+        boolean after(String s, String other) {
+            return s.equalsIgnoreCase(other);
+        }
+    }
+
+    private static class Equals {
+        @BeforeTemplate
+        boolean before(String s, String other) {
+            return StringUtils.equals(s, other);
+        }
+
+        @AfterTemplate
+        boolean after(String s, String other) {
+            return Objects.equals(s, other);
+        }
+    }
+
+    private static class IndexOfAny {
+        @BeforeTemplate
+        int before(String s, String search) {
+            return StringUtils.indexOfAny(s, search);
+        }
+
+        @AfterTemplate
+        int after(String s, String search) {
+            return IntStream.range(0, s.length())
+                    .filter(i -> search.indexOf(s.charAt(i)) >= 0)
+                    .findFirst()
+                    .orElse(-1);
+        }
+    }
+
+    private static class IsAlphanumericSpace {
+        @BeforeTemplate
+        boolean before(String s) {
+            return StringUtils.isAlphanumericSpace(s);
+        }
+
+        boolean after(String s) {
+            return s.matches("^[a-zA-Z0-9\\s]*$");
```

Comment:
I suspect this isn't correct with respect to Unicode characters (i.e. non ASCII characters). I suggest to instead use a `Stream` based on `String#chars()` like you do in `IsAlpha`.